src/lib/util.ts

```diff
@@ -0,0 +1,369 @@
+import {
+	Channel,
+	MediaLayer,
+	NextUpMedia,
+	NextUp,
+	LayerContentType,
+	LayerBase,
+	TransitionObject,
+	Transition,
+	State
+} from './api'
+import * as _ from 'underscore'
+import { Mixer } from './mixer'
+import { AMCPCommandWithContext, AMCPCommandVOWithContext, DiffCommands } from './casparCGState'
+import { Command as CommandNS } from 'casparcg-connection'
+import { InternalLayer, InternalState } from './stateObjectStorage'
+
+export function frames2Time(frames: number, fps = 25): number {
```

Comment:
I was thinking if it would be better to make the function argument `fps` be required, ie `fps: number | undefined` (then set the default value inside the function). That way we can ensure it's not forgotten when used in the future.